util.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+main() {
+  case $1 in
```

Comment:
Can we explicitly convert the first arg to a named variable?  Otherwise, errors may occur that are harder to understand, like `unbound variable $1`.